### ISSUE TITLE
CSIP62 Refer to content folders instead of representations

### ISSUE THREAD

**Issue by koit** [DILCISBoard/E-ARK-CSIP#436]
The current definition of CSIP62 reads:

```xml
<requirement ID="CSIP62" REQLEVEL="SHOULD" RELATEDMAT="VocabularyContentInformationTypeSpecification" EXAMPLES="fileSecExample1 fileSecExample2">
  <description>
    <head>Specific content type</head>
    <p xmlns="http://www.w3.org/1999/xhtml">An added attribute which describes the specific content information type specification used for the transferred content. The attribute is mandatory when the file group catagorisation value is "Representations". The vocabulary will evolve under the curation of the DILCIS Board as additional content information type specifications are developed.</p>
    <dl xmlns="http://www.w3.org/1999/xhtml">
      <dt>METS XPath</dt><dd>fileSec/fileGrp/@csip:CONTENTINFORMATIONTYPE</dd>
      <dt>Cardinality</dt><dd>1..1</dd>
    </dl>
  </description>
</requirement>
```

It might be irrelevant now that the folder structure is optional, but I have understood that the folder "representations" (i.e. the root folder for representations) should not contain any files, but only subfolders for individual representations, and there should be nested fileGrp elements (a parent for "representations" and children for "rep1", "rep2" etc.). And according to CSIP64:

>The value in the `@USE` is the name of the whole folder structure to the data, e.g "Documentation", "Schemas", "Representations/preingest" or "Representations/submission/data".

Consequently, there should be no fileGrp where the current wording of CSIP62 would apply. Unless we deliberately want to provide for declaring a CONTENTINFORMATIONTYPE for all representations at once, but then again, this can already be done in `mets/@csip:CONTENTINFORMATIONTYPE`.

Also, my understanding is that a representation is an instance of the core thing (i.e. payload data + its metadata) and multiple representations inside a package should be instances of that same thing, not of entirely different things. E.g. [Principle 3.4](http://earkcsip.dilcis.eu/#principle-34) states:

>As technologies evolve and get obsolete, data and metadata is constantly updated /…/, therefore creating new versions or representations of the data and metadata.

Thus, different content types (e.g. ERMS and SIARD2) should not be in different representations but in different folders of the same representation. And each representation can have its own METS.xml with its own fileGrp elements. So CSIP62 should not talk about representations, but content files.

In conclusion, I think CSIP62 should refer to the folders of actual payload files, which are located under individual representations, so the description should be something like:

>An attribute added by CSIP to describe the content information type of the fileGrp, to allow packages with mixed content. The attribute is mandatory for fileGrp elements that group actual content files.
>
>The values MUST be from the vocabulary `VocabularyContentInformationTypeSpecification` (which will evolve under the curation of the DILCIS Board as additional content information type specifications are developed).

Note that there is no `VocabularyContentInformationTypeSpecification` in DILCISVocabulariesIP.xml, it is currently called `VocabularyContentTypeSpecification`.

Related to #435.
